### PR TITLE
calculator unit tests

### PR DESCRIPTION
**Pull request by szaszm**
Add some unit tests

### CODE REVIEW

**Comment by szaszm** [comment]
#9

File: maths/test_maths.py [hunk 1]
```diff
@@ -190,14 +193,13 @@ class TestRSA(unittest.TestCase):
...
-    # def test_key_generator(self): # this test takes a while!
-    #     for i in range(100):
-    #         print("step {0}".format(i))
-    #         n, e, d = generate_key(26)
-    #         data = 2
-    #         en = encrypt(data, e, n)
-    #         dec = decrypt(en, d, n)
-    #         self.assertEqual(data,dec)
+
+    def test_key_generator(self): # this test takes a while!
```
Comment (csutorasr) [inline maths/test_maths.py]: Does this take a while without 100 repetitions?

File: calculator/math_parser.py [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
     Code:
-        |   exp = "2452 * (3 * 6.5 + 1) * 6 / 235"
+        |   exp = "2452*(3*6.5+1)*6/235"
         |   print("Expression:", exp)
         |   print("Parsed expression:", mp.parse(exp))
         |   print("Evaluation result:", mp.evaluate(exp))
 
     Output:
-        |   Expression: 2452 * (3 * 6 + 1) * 6 / 235
+        |   Expression: 2452*(3*6+1)*6/235
```
Comment (csutorasr) [inline calculator/math_parser.py]: I think this is harder to read.

File: maths/test_maths.py (hunk 1 above)
Comment (szaszm) [reply maths/test_maths.py]: the loop was removed

File: calculator/math_parser.py (hunk 2 above)
Comment (szaszm) [reply calculator/math_parser.py]: It is, but the parser does not accept strings containing spaces so it's more clear

**Comment by szaszm** [comment]
I replied to the comments and I think changes are not needed.

File: maths/test_maths.py (hunk 1 above)
Comment (csutorasr) [reply maths/test_maths.py]: This is not an answer to the question. The comment is not needed if it doesn't take a while anymore.

Comment (szaszm) [reply maths/test_maths.py]: It still takes a couple of seconds

Comment (csutorasr) [reply maths/test_maths.py]: Not a while, but OK.

**Pull request merged**